src/assets/stylesheets/shared/_custom-mixins.scss

```diff
@@ -0,0 +1,99 @@
+@import 'bootstrap/mixins';
```

Comment:
as i understand from the code there is a webpack plugin that appends the relative path to the bootstrap sass as it walks the dependency tree